themes/default/content/blog/announcing-crossguard-preview/index.md

```diff
@@ -4,7 +4,7 @@ date: 2019-12-02
 meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution that enforces custom infrastructure policies, is available for all users to preview."
 meta_image: crossguard.svg
 authors: ["erin-krengel"]
-tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News", "Features"]
+tags: ["policy-as-code", "new-features"]
```

Comment:
Was it intentional to remove Pulumi-news here (even though the tag still exists)?

Reply:
should this just be features? since they wont stay "new" for very long, or maybe that's not a concern?

Reply:
@lukehoban yes. I've reserved `pulumi-news` for company news (funding, SOC2, etc.) and focused `new-features` for... new features.

@susanev that's a good point. I usually view blogs as a "point-in-time" medium. Put another way, I could make a case for keeping `new-features` since, at the time the blog post was authored, the feature was, in fact, new. But, I like shorter tags in general, so I'll adjust to just `features`.